acceptance/tests/facts/windows.rb

```diff
@@ -88,8 +88,13 @@
   end
 
   step "Ensure the identity fact resolves as expected"
+  # Regular expression to validate the username from facter in the form of '<domain>\<username>'
+  # Reference - https://msdn.microsoft.com/en-us/library/bb726984.aspx
+  # - The domain name can be any character or empty
+  # - Must contain a backslash between the domain and username
+  # - Username must be at least one character and not contain the following charaters; " / \ [ ] : ; | = , + * ? < >
   expected_identity = {
-                        'identity.user'       => /.*\\cyg_server/,
+                        'identity.user'       => /.*\\[^\\\/\"\[\]:|<>+=;,?*@]+$/,
```

Comment:
I'm ok with leaving it for a later ticket to match the actual value. It's pretty unlikely we'll get a value, but miss something because that regex was too broad.